core-services/prow/02_config/_plugins.yaml

```diff
@@ -3240,6 +3244,9 @@ lgtm:
   - openshift-priv/azure-disk-csi-driver
   - openshift/azure-disk-csi-driver
   review_acts_as_lgtm: true
+- repos:
+  - openshift/cloud-network-config-controller
+  review_acts_as_lgtm: true
```

Comment:
what is the implication of this tag? If someone finishes the review, does that automatically LGTM the PR?

Reply:
Not sure, I just ran `make new-repo`... @alvaroaleman , could you fill us in?

Reply:
It means that the github review (when you click review -> approve) will count as lgtm